sdk/appconfiguration/app-configuration/package.json

```diff
@@ -131,7 +131,7 @@
     "rollup-plugin-terser": "^5.1.1",
     "sinon": "^9.0.2",
     "ts-node": "^8.3.0",
-    "typescript": "~3.9.3",
+    "typescript": "4.1.2",
```

Comment:
We normally don't use strict versions. Any reason why we might use a strict version here?

Reply:
(same with the other instances of a strict version in this PR)

Reply:
Typescript doesn't follow semver rules, and can break things in between minor versions as well.
So, we need to pin the version for typescript.

Reply:
Alright! Thank you 🌞